resources/metadata/mavenBuild.yaml

```diff
@@ -236,6 +236,8 @@ spec:
         type: piperEnvironment
         params:
           - name: custom/buildSettingsInfo
+          - name: artifacts
```

Comment:
Since the PR to the `golangBuild` has been already merged and it has [the same configuration in the metadata file](https://github.com/SAP/jenkins-library/pull/3749/files#diff-f30b9c6afbefb2598630071681fb9a95d15e0234cbe608320c5c1246a071e97aR243-R244), does it mean that getters/setters have been already added, or that the `golangBuild` step contains a potential bug?

Reply:
@CCFenner You are probably best to answer @pbusko's question above. I also wondered why the getters/setters have not been relevant for the `golang` step. But maybe I missed something.